### ISSUE TITLE
ERROR PARSING CRAFT FILE FROM DIRECTORY

### ISSUE THREAD

**Issue by LuckySparkle** [msummers123/Movecraft-3#258]
I have a custom made .craft file but movecraft just responds the error 
***ERROR PARSING CRAFT FILE FROM DIRECTORY: /home/minecraft/spigot/plugin/Movecraft/types/korvette.craft***

I already found another Issue with the same Problem
https://github.com/msummers123/Movecraft-3/issues/178
and tryed to fix my .craft file but it didn't worked out... 

file can be found here
hoerth.eu/korvette.rar

name: Korvette
maxSize: 3500
minSize: 2500
allowedBlocks:
  - 1
  - 4
  - 5
  - 17
  - 20
  - 22
  - 23
  - 24
  - 26
  - 29
  - 30
  - 33
  - 35
  - 41
  - 42
  - 43
  - 44
  - 45
  - 47
  - 48
  - 49
  - 50
  - 53
  - 54
  - 57
  - 58
  - 61
  - 62
  - 63
  - 64
  - 65
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 75
  - 76
  - 77
  - 85
  - 86
  - 87
  - 88
  - 89
  - 91
  - 92
  - 95
  - 96
  - 101
  - 102
  - 103
  - 107
  - 108
  - 109
  - 112
  - 113
  - 114
  - 116
  - 117
  - 118
  - 123
  - 124
  - 125
  - 126
  - 128
  - 130
  - 131
  - 132
  - 133
  - 134
  - 135
  - 136
  - 139
  - 140
  - 143
  - 145
  - 146
  - 147
  - 148
  - 151
  - 155
  - 156
  - 158
  - 160
  - 162
  - 163
  - 164
  - 167
  - 168
  - 169
  - 170
  - 171
  - 176
  - 177
  - 183
  - 184
  - 185
  - 186
  - 187
  - 188
  - 189
  - 190
  - 191
  - 192
  - 193
  - 194
  - 195
  - 196
  - 197
  - 214
  - 215
  - 218
  - 323
  - 324
  - 321
  - 330
  - 355
  - 354
  - 379
  - 380
  - 425
  - 327
  - 428
  - 429
  - 430
  - 416
  - 431
blockedByWater: false
speed: 10.0
tryNudge: false
flyblocks:
  35:
    - 20.0
    - 100.0

**Comment by Iridus** [comment]
I'm also having the same problem as you. I figure it's better to reply to you rather than making another post so we don't have two posts opened on the same issue at the same time. Here's my file: [https://www.dropbox.com/s/xb6g3fy4vbud9kt/Tank2.craft?dl=0](url). I've ran both my .craft and yours through a yaml validator and they seem to be ok, so I don't know what is the issue.

**Comment by keough99** [comment]
Do you have a - before the block ids?

**Comment by LuckySparkle** [comment]
Yes I have, I made a Screenshot from the opened file in notepad++
I think you can see everything important there
[https://hoerth.eu/craftissue.jpg](https://hoerth.eu/craftissue.jpg)

Also you can download the complete file here
[https://hoerth.eu/korvette.rar](https://hoerth.eu/korvette.rar)

**Comment by pistols12** [comment]
The file you linked is not properly spaced

**Comment by pistols12** [comment]
Let me elaborate. on your allowed blocks: 
you had all the allowed blocks 2 spaces from the left hand side. Like this:

Allowed Blocks:
  - 1
  - 2

it needed to be 4 spaces from left so that the file would read it all correctly, like so:

Allowed Blocks:
    - 1
    - 2

I fixed 100+ lines in your file, and am sending it back to you. I've added some extra nodes for extra customization, as well as cleaned up some other areas of the file!

please give me feedback!  

Your Movecraft Geek,

CaptaingibbonsSR III

Link to fixed file: https://www.dropbox.com/s/a3v846x3is3r2r1/korvette.craft?dl=0

**Comment by pistols12** [comment]
P.S.  For all parsing issues, please refer to this site to help with finding where they are: 

http://yaml-online-parser.appspot.com/


As long as your spacing is correct, this will help you find any indentations, tabs, etc etc. It's a great tool, also, if you have an issue it cannot find, open up two tabs, but a known good file in one window of a yaml parser, and the problem child in another, that is how i found this issue here, with your spacing and so on. 

If you need any tips just ask, also PLEASE CLOSE THIS ISSUE WHEN VERIFIED FIXED.

**Comment by pistols12** [comment]
Idridus, I am now getting to your file.

**Comment by pistols12** [comment]
Iridus, the drop box link you gave is no longer working. Make a new Issue Ticket please with a new link
